plugins/feeds/public/hybrid_analysis.py

```diff
@@ -84,7 +84,8 @@ def analyze(self, item):
                     f_hyb.active_link_to(new_host, 'C2', self.name)
                     logging.debug(domain)
 
-                    new_host.add_context({'source': self.name, 'contacted by': f_hyb})
+                    new_host.add_context({'source': self.name,
+                                          'contacted by': f_hyb})
```

Comment:
No spaces in context keys, otherwise it makes search impossible. Please use `contacted_by`.